maquetado/scss/detail.scss

```diff
@@ -10,19 +10,27 @@
 
 .photo-detail {
   height: 368px;
+  margin-right: 52px;
   width: 261px;
 }
 
 .information {
   display: flex;
+  flex: 1;
   flex-flow: column wrap;
-  margin: 0 52px;
 }
 
+
 .card-description {
-  margin: 18px 6px;
+  margin-bottom: 36px;;
+}
+
+.card-description:first-child {
```

Comment:
Move these styles border-bottom and padding to inside .card-description definition , look that you can use this selector width .card-description